user-interface/src/search/SearchScreen.tsx

```diff
@@ -13,25 +13,33 @@ import { courtSorter } from '@/data-verification/DataVerificationScreen';
 import Alert, { UswdsAlertStyle } from '@/lib/components/uswds/Alert';
 import './SearchScreen.scss';
 import ComboBox, { ComboOption } from '@/lib/components/combobox/ComboBox';
-import { isValidSearchPredicate, SearchResults } from '@/search-results/SearchResults';
+import SearchResults, { isValidSearchPredicate } from '@/search-results/SearchResults';
 import { SearchResultsHeader } from './SearchResultsHeader';
 import { SearchResultsRow } from './SearchResultsRow';
 import { useGlobalAlert } from '@/lib/hooks/UseGlobalAlert';
 import DocumentTitle from '@/lib/components/cams/DocumentTitle/DocumentTitle';
 import { MainContent } from '@/lib/components/cams/MainContent/MainContent';
+import Button, { ButtonRef } from '@/lib/components/uswds/Button';
+import { UswdsButtonStyle } from '../lib/components/uswds/Button';
```

Comment:
Looks like these are all coming from the same file.  We should add UswdsButtonStyle to the line above it and also try to use the '@' whenever we are going up to a parent folder or beyond.